### PR TITLE
content-watcher: some fixes

### PR DESCRIPTION
**Pull request by aramikm**
# Problem

 Related to #631

# Changes
- renamed and added some documentation's around confusing payloads
- changed `PUT /v1/search` to `POST /v1/search`
- Fixed using of `webhookUrl` in search payload

# Impact
- Going to open a new PR on Social app template and apply impacted changes

### CODE REVIEW

File: apps/content-watcher/src/crawler/crawler.service.ts [hunk 1]
```diff
@@ -30,27 +30,33 @@ export class CrawlerService extends BaseConsumer {
...
+      if (!upperBoundBlock) {
+        upperBoundBlock = latestBlock;
+        this.logger.debug(`No starting block specified; starting from end of chain at block ${upperBoundBlock}`);
       }
+      upperBoundBlock = Math.min(upperBoundBlock, latestBlock);
+      // eslint-disable-next-line no-param-reassign
+      job.data.upperBoundBlock = upperBoundBlock;
       // Make sure blockCount is not longer than the current chain length
-      if (job.data.blockCount >= startBlock) {
+      if (job.data.blockCount >= upperBoundBlock) {
```
Comment (mattheworris) [inline apps/content-watcher/src/crawler/crawler.service.ts:46]: I like the clarity of the new naming.

**Review by mattheworris — APPROVED** [review]
- Read through code
Looks good!
🚢 it!

**Review by saraswatpuneet — APPROVED** [review]
Neat! Lgtm

File: apps/content-watcher/src/controllers/v1/search.controller.ts [hunk 2]
```diff
@@ -12,7 +12,10 @@ export class SearchControllerV1 {
 
   @Post()
   @HttpCode(HttpStatus.OK)
-  @ApiOperation({ summary: 'Search for DSNP content by id, start/end block, and filters' })
+  @ApiOperation({
+    summary:
+      'Search for DSNP content by id and filters, starting from `upperBound` block and going back for `blockCount` number of blocks',
```
Comment (JoeCap08055) [inline apps/content-watcher/src/controllers/v1/search.controller.ts:17]: Just a note; `upperBoundBlock` is not required; will default to latest block (as of when the search job actually begins to process) if not provided.

**Pull request merged**